dropwizard-async-http-client/src/main/java/smartthings/dw/asynchttpclient/AsyncHttpClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package smartthings.dw.asynchttpclient;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AsyncHttpClientConfig {
+	private Map<String, Object> properties = new HashMap<>();
```

Comment:
There are a lot of other properties that'd be interesting to test under load, but I figure we can start with those simple ones.